src/eslint-config-adeira/README.md

```diff
@@ -80,6 +80,8 @@ It tries to detect files to lint because it's highly inefficient to test all the
 
 Please note: this Eslint runner not only runs all the tests much faster but it also performs automatic fixes. This is currently no-opt.
 
+You can also suppress warnings in report by `--no-warnings` option. This is not to make you ignore warnings, rather an utility to easily filter out problems in changed code while you are in process of updating your legacy codebase with a lot of warnings. 
```

Comment:
I don't understand this explanation. For me, it's exactly for ignoring the warnings. It would be nice to show the intended usage maybe since I cannot imagine how is this going to be used.

Reply:
ok, I will rethink how to formulate this 🤔